code/modules/migrations/mysql/mysql_controller.dm

```diff
@@ -31,7 +31,10 @@ CREATE TABLE IF NOT EXISTS [TABLE_NAME] (
 
 	var/list/rows=list()
 	while(query.NextRow())
-		rows += list(query.item)
```

Comment:
THIS was the bad line that was causing the MySQL migrations to runtime and as such not work. I have no idea why the fuck this doesn't work but my method does work so whatever.